src/components/molecules/SpaceEditForm/SpaceEditForm.tsx

```diff
@@ -353,26 +353,26 @@ export const SpaceEditForm: React.FC<SpaceEditFormProps> = ({ space }) => {
           {BACKGROUND_IMG_TEMPLATES.includes(
             space.template as VenueTemplate
           ) && (
-            <AdminSidebarAccordion title="Map background">
-              <BackgroundSelect
-                isLoadingBackgrounds={isLoadingBackgrounds}
-                mapBackgrounds={mapBackgrounds}
-                venueName={space.name}
-                spaceSlug={space.slug}
-                worldId={space.worldId}
-                venueId={space.id}
-              />
-              {errorFetchBackgrounds && (
-                <>
-                  <div>
-                    The preset map backgrounds could not be fetched. Please,
-                    refresh the page or upload a custom map background.
-                  </div>
-                  <div>Error: {errorFetchBackgrounds.message}</div>
-                </>
-              )}
-            </AdminSidebarAccordion>
-          )}
+              <AdminSidebarAccordion title="Map background">
```

Comment:
[Q] I don't notice code change, how does this happen? Isn't reformat on commit supposed to produce the same indent?